docs/localdatetime.md

```diff
@@ -0,0 +1,1242 @@
+
+```javascript
+Temporal.Absolute.from('2020-08-05T20:06:13+09:00[+09:00]');
+// OR
+Temporal.Absolute('2020-08-05T20:06:13+09:00').toLocalDateTime('+09:00', 'iso8601');
+```
+
+Note that using `Temporal.LocalDateTime` with an offset time zone will not adjust for Daylight Savings Time or other time zone changes. Therefore, using offset time zones with this type is relatively unusual. Instead of using an offset time zone, it may be more straightforward for most use cases to simply use `Temporal.DateTime` and/or `Temporal.Absolute`.
+
+The `overflow` option works as follows:
+
+- In `constrain` mode (the default), any out-of-range values are clamped to the nearest in-range value.
+- In `reject` mode, the presence of out-of-range values will cause the function to throw a `RangeError`.
+
+Additionally, if the result is earlier or later than the range of dates that `Temporal.DateTime` can represent (approximately half a million years centered on the [Unix epoch](https://en.wikipedia.org/wiki/Unix_time)), then this method will throw a `RangeError` regardless of `overflow`.
+
+> **NOTE**: Although Temporal does not deal with leap seconds, dates coming from other software may have a `second` value of 60.
+> In the default `constrain` mode and when parsing an ISO 8601 string, this will be converted to 59.
+> In `reject` mode, this function will throw, so if you have to interoperate with times that may contain leap seconds, don't use `reject`.
+
+If the input contains a time zone offset, in rare cases it's possible for those values to conflict for a particular local date and time.
+For example, this could happen for future summertime events that were stored before a country permanently abolished DST.
+If the time zone and offset are in conflict, then the `offset` option is used to resolve the conflict:
+
+- `'use'`: Evaluate date/time values using the time zone offset if it's provided in the input.
+  This will keep the absolute time unchanged even if local time will be different than what was originally stored.
+- `'ignore'`: Never use the time zone offset provided in the input. Instead, calculate the offset from the time zone.
+  This will keep local time unchanged but may result in a different absolute time than was originally stored.
+- `'prefer'`: Evaluate date/time values using the offset if it's valid for this time zone.
+  If the offset is invalid, then calculate the offset from the time zone.
+  This option is rarely used when calling `from()`.
+  See the documentation of `with()` for more details about why this option is used.
+- `'reject'`: Throw a `RangeError` if the offset is not valid for the provided date and time in the provided time zone.
+
+The primary reason to use the `offset` option is when parsing values which were saved before a change to that time zone's time zone definition.
```

Comment:
OK I'll take a look at that page and see where this could fit.

Reply:
Do you mean moving the `disambiguation`-related content, the `offset`-related content, or both?

I ask because `offset` is only relevant to LDT (because no other type parses values with both offset and IANA zone), while `disambiguation` applies to many other types.

BTW, today I revamped the ambiguity page and will check it in soon. I updated the disambiguation samples on that page to include LDT and made it into more of an "About Time Zones and Temporal" page to include some generalized content that might be useful for newbies to this space. Will check in once I figure out if `offset` content should go there or not.

Reply:
I meant moving the `offset` section, lines 138 through 170, I think it fits thematically on the ambiguity page, even though it only applies to LocalDateTime. I think it could go in its own section on that page. My comment was because I think the section goes into too much detail for reference documentation and makes it hard to scan (which is, to be fair, a problem with some of the other docs I wrote as well) but I did think it was important enough that it shouldn't just be deleted.

Can you check in the ambiguity page as a separate PR so that this one doesn't have to block on it?

Reply:
> I meant moving the offset section, lines 138 through 170,
OK I'll take a look. 

> I think it fits thematically on the ambiguity page, even though it only applies to LocalDateTime. I think it could go in its own section on that page.

Good, because I already did this. ;-) This is already done and checked in-- it's the bottom two sections in the revised ambiguity page in this PR. 

> Can you check in the ambiguity page as a separate PR so that this one doesn't have to block on it?
I actually finished the updated ambiguity page already-- it's already checked into this PR and I don't anticipate any more updates to it. Do you want me to pull it out of this PR? I was hoping to leave it in because the LDT content will point to a new section in the ambiguity page.

Reply:
OK, removed  about 65 LOC from the `from` docs, mostly `offset` drilldown content/samples and `disambiguation` samples.  The remaining `offset` content now points readers to the new section in the ambiguity page for `offset` drilldown details.